pos_restricted_customer_list/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    available_in_pos = fields.Boolean(
+        string="Available for POS",
+        default=False,
```

Comment:
Shouldn't this be `True`? What happens when a cashier creates a customer from the frontend, does it become unavailable later?

Reply:
There was no default value in the v8 module (so it was `False`), i just made it explicit here. I prefer not to change the behavior in the migration PR.

Reply:
Ok, but, what happens in the case I mentioned?

Reply:
It will be available in the POS where the cashier created it but not in others.

Reply:
Until that pos is reloaded. Imagine I create a customer for a ticket and... oops, I accidentally pressed F5... now I don't have the customer anymore and probably an error raises for the ticket I had opened

Reply:
It disappears only after closing and posting the entries so no error. But I guess it's not intuitive. I would say that the `default` in the field's definition must remain `False` but `available_in_pos` must be set to `True` when creating a partner from a POS. Wyt?

Reply:
Done